src/test/App.test.tsx

```diff
@@ -0,0 +1,13 @@
+// src/__ tests __/App.test.tsx
+import { render } from "@testing-library/react"
+import App from "../App"
+import '@testing-library/jest-dom'
+
+test('demo', () => {
+    expect(true).toBe(true)
+})
+
+test("Renders the main page", () => {
+    render(<App />)
+    expect(true).toBeTruthy()
```

Comment:
**chore** 🧹
Please assert the document contains the word "Recette".

<sub><sup>I'm using [conventional comments](https://conventionalcomments.org/) formatting to improve comments readability and reader's expectations in this PR.</sup></sub>